en/includes/guides/organization-management/email-domain-based-organization-discovery.md

```diff
@@ -27,8 +27,9 @@ and user Bob and Ben, with emails `bob@gcmart.com` and `ben@glory.com` should be
 ### Using the Console
 
 1. Login to the organization (root) from the {{ product_name }} Console.
-2. On the {{ product_name }} Console, go to **Login & Registration**, and click **Email Domain Discovery** under **Organization Settings**.
+2. On the {{ product_name }} Console, go to **Login & Registration**, and click **Organization Discovery** under **Organization Settings**.
 3. Turn on the toggle to enable email domain based organization discovery.
```

Comment:
Explain what will happen when toggle is enabled

Reply:
Updated with [8d1e3e2](https://github.com/wso2/docs-is/pull/4923/commits/8d1e3e23d53b145b04e82f8fcc6506ff77ae6b54)